<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1195</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="OpenBinFile">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OpenBinFile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ReadBinFile">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ReadBinFile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CloseBinFile">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CloseBinFIle</string>
    </property>
   </widget>
   <widget class="QListWidget" name="ShowBinaryFile">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>951</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="RefreshPorts">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>RefreshPorts</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenPort">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OpenPort</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ListPorts">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ClosePort">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ClosePort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SendBinary">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SendBinary</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="SendingProgress">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QListWidget" name="ShowSendingBinary">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>160</y>
      <width>791</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1195</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
